src/main/scala/no/ndla/articleapi/model/domain/Article.scala

```diff
@@ -29,6 +29,7 @@ case class Article(id: Option[Long],
                    metaImageId: Option[String],
                    created: Date,
                    updated: Date,
+                   updatedBy: String,
```

Comment:
Når vi introduserer et nytt feilt i domene artikkelen må vi også sørge for at eksisterende dokumenter i databasen har et tilsvarende felt, hvis ikke vil json -> case class-konvertering krasje fordi feltet "updatedBy" ikke finnes i database dokumentet. Måten vi har løst dette på før er å lage [migration script](https://github.com/NDLANO/image-api/blob/master/src/main/scala/db/migration/V2__RemoveFullFromImagePath.scala)